### ISSUE TITLE
TB 91.0beta6 - No longer able to move a message by dropping it on a quick folder tab 

### ISSUE THREAD

**Issue by cbroult** [RealRaven2000/QuickFolders#189]
This issue seems to relate to  #114

### Steps:
1. Click on a message in the message list pane
2. Hold left mouse click to drag the message 
3. Drop the the message icon on a quick folder tab

### Expected
* The message is moved

### Actual
* Nothing seems to happen
* The following exception is logged:
```
Exception in QuickFolders.Util.moveMessages, step 7:
TypeError: cs.CopyMessages is not a function quickfolders-util.js:1057:13
```

### Note

It is still possible to move a message using the left side folder hierarchy pane.
In that case the QuickFilter dialog box still opens after the message was moved.

### Console log

```
toolbarDragObserver:DnD dragenter QuickFolders-Toolbar quickfolders.js:1019:12
toolbarDragObserver:DnD dragenter QuickFolders-Folders-Pane quickfolders.js:1019:12
toolbarDragObserver:DnD dragenter QuickFolders-FoldersBox quickfolders.js:1019:12
toolbarDragObserver:DnD dragenter quickfolders.js:1019:12
QuickFolders 19:25:21.13  [9359 ms]   
 dragSession =  
XPCWrappedNative_NoHelper { QueryInterface: QueryInterface(), invokeDragSession: invokeDragSession(), getCurrentSession: getCurrentSession(), startDragSession: startDragSession(), startDragSessionForTests: startDragSessionForTests(), endDragSession: endDragSession(), suppress: suppress(), unsuppress: unsuppress(), DRAGDROP_ACTION_NONE: 0, DRAGDROP_ACTION_COPY: 1, … }
quickfolders-util.js:1057:13
QuickFolders 19:25:21.16  [3 ms]   
 Exception in QuickFolders.Util.moveMessages, step 7:
TypeError: cs.CopyMessages is not a function quickfolders-util.js:1057:13
QuickFolders 19:25:21.16  [0 ms]   
 buttonDragObserver.drop DropTarget = toolbarbutton0  Target Folder:00.offers quickfolders-util.js:1057:13
QuickFolders:Exception in drop -createFilterAsync(Inbox, 00.offers, no Messages returned!)... TypeError: messageIdList is null
```

### Troubleshooting Information

```
  Application Basics

    Name: Thunderbird
    Version: 91.0b6
    Build ID: 20210802205227
    Distribution ID:

    Update Channel: beta
    User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:91.0) Gecko/20100101 Thunderbird/91.0
    OS: Windows_NT 10.0 19043

    Launcher Process: Enabled
    Multiprocess Windows: 0/0
    Fission Windows: 0/0
              Disabled by default
    Remote Processes: 5
    Enterprise Policies: Inactive
    Google Location Service Key: Missing
    Google Safebrowsing Key: Missing
    Mozilla Location Service Key: Missing
    Safe Mode: false

  Mail and News Accounts
    REMOVED
  Calendar Settings
     REMOVED
  Crash Reports for the Last 3 Days
    
  Remote Processes

    Type: Count

      Web Content: 3 / 8
      Extension: 1
      GPU: 1

  Add-ons

      Name
      Type
      Version
      Enabled
      ID

        French spelling dictionary
        dictionary
        6.3.1webext
        true
        fr-dicollecte@dictionaries.addons.mozilla.org

        German  dictionary (de_DE)
        dictionary
        20180701.1webext
        true
        de_DE@dicts.j3e.de

        Amazon.com
        extension
        1.1
        true
        amazondotcom@search.mozilla.org

        Bing
        extension
        1.0
        true
        bing@search.mozilla.org

        DuckDuckGo
        extension
        1.0
        true
        ddg@search.mozilla.org

        Google
        extension
        1.0
        true
        google@search.mozilla.org

        quickFilters
        extension
        5.3.1
        true
        quickFilters@axelg.com

        QuickFolders
        extension
        5.6.6
        true
        quickfolders@curious.be

        Quicktext
        extension
        4.0
        true
        {8845E3B3-E8FB-40E2-95E9-EC40294818C4}

        Thunderbird Conversations
        extension
        3.2.28
        true
        gconversation@xulforum.org

        Wikipedia (en)
        extension
        1.0
        true
        wikipedia@search.mozilla.org

  Security Software

    Type: Name

      Antivirus: Microsoft Defender Antivirus
      Antispyware:
      Firewall: Windows Firewall

  Graphics

      Features
      Compositing: WebRender
      Asynchronous Pan/Zoom: wheel input enabled; scrollbar drag enabled; keyboard enabled; autoscroll enabled; smooth pinch-zoom enabled
      WebGL 1 Driver WSI Info: EGL_VENDOR: Google Inc. (adapter LUID: 000000000001081c)
EGL_VERSION: 1.5 (ANGLE 2.1.14226 git hash: 31a43497be09)
EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_EXT_device_query EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync
EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control
IsWebglOutOfProcessEnabled: 1
      WebGL 1 Driver Renderer: Google Inc. -- ANGLE (AMD Radeon RX 5700 Direct3D11 vs_5_0 ps_5_0)
      WebGL 1 Driver Version: OpenGL ES 2.0.0 (ANGLE 2.1.14226 git hash: 31a43497be09)
      WebGL 1 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_color_buffer_float_rgb GL_CHROMIUM_color_buffer_float_rgba GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_half_float GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_WEBGL_video_texture
      WebGL 1 Extensions: ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_float_blend EXT_frag_depth EXT_shader_texture_lod EXT_sRGB EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_element_index_uint OES_fbo_render_mipmap OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context
      WebGL 2 Driver WSI Info: EGL_VENDOR: Google Inc. (adapter LUID: 000000000001081c)
EGL_VERSION: 1.5 (ANGLE 2.1.14226 git hash: 31a43497be09)
EGL_EXTENSIONS: EGL_EXT_create_context_robustness EGL_ANGLE_d3d_share_handle_client_buffer EGL_ANGLE_d3d_texture_client_buffer EGL_ANGLE_surface_d3d_texture_2d_share_handle EGL_ANGLE_query_surface_pointer EGL_ANGLE_window_fixed_size EGL_ANGLE_keyed_mutex EGL_ANGLE_surface_orientation EGL_ANGLE_direct_composition EGL_NV_post_sub_buffer EGL_KHR_create_context EGL_EXT_device_query EGL_KHR_image EGL_KHR_image_base EGL_KHR_gl_texture_2D_image EGL_KHR_gl_texture_cubemap_image EGL_KHR_gl_renderbuffer_image EGL_KHR_get_all_proc_addresses EGL_KHR_stream EGL_KHR_stream_consumer_gltexture EGL_NV_stream_consumer_gltexture_yuv EGL_ANGLE_flexible_surface_compatibility EGL_ANGLE_stream_producer_d3d_texture EGL_ANGLE_create_context_webgl_compatibility EGL_CHROMIUM_create_context_bind_generates_resource EGL_CHROMIUM_sync_control EGL_EXT_pixel_format_float EGL_KHR_surfaceless_context EGL_ANGLE_display_texture_share_group EGL_ANGLE_display_semaphore_share_group EGL_ANGLE_create_context_client_arrays EGL_ANGLE_program_cache_control EGL_ANGLE_robust_resource_initialization EGL_ANGLE_create_context_extensions_enabled EGL_ANDROID_blob_cache EGL_ANDROID_recordable EGL_ANGLE_image_d3d11_texture EGL_ANGLE_create_context_backwards_compatible EGL_KHR_create_context_no_error EGL_KHR_reusable_sync
EGL_EXTENSIONS(nullptr): EGL_EXT_client_extensions EGL_EXT_platform_base EGL_EXT_platform_device EGL_ANGLE_platform_angle EGL_ANGLE_platform_angle_d3d EGL_ANGLE_platform_angle_d3d11on12 EGL_ANGLE_device_creation EGL_ANGLE_device_creation_d3d11 EGL_ANGLE_experimental_present_path EGL_KHR_client_get_all_proc_addresses EGL_KHR_debug EGL_ANGLE_explicit_context EGL_ANGLE_feature_control
IsWebglOutOfProcessEnabled: 1
      WebGL 2 Driver Renderer: Google Inc. -- ANGLE (AMD Radeon RX 5700 Direct3D11 vs_5_0 ps_5_0)
      WebGL 2 Driver Version: OpenGL ES 3.0.0 (ANGLE 2.1.14226 git hash: 31a43497be09)
      WebGL 2 Driver Extensions: GL_ANGLE_base_vertex_base_instance GL_ANGLE_client_arrays GL_ANGLE_copy_texture_3d GL_ANGLE_depth_texture GL_ANGLE_explicit_context GL_ANGLE_explicit_context_gles1 GL_ANGLE_framebuffer_blit GL_ANGLE_framebuffer_multisample GL_ANGLE_get_tex_level_parameter GL_ANGLE_instanced_arrays GL_ANGLE_lossy_etc_decode GL_ANGLE_memory_size GL_ANGLE_multi_draw GL_ANGLE_multiview_multisample GL_ANGLE_pack_reverse_row_order GL_ANGLE_program_cache_control GL_ANGLE_provoking_vertex GL_ANGLE_request_extension GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ANGLE_texture_multisample GL_ANGLE_texture_usage GL_ANGLE_translated_shader_source GL_CHROMIUM_bind_generates_resource GL_CHROMIUM_bind_uniform_location GL_CHROMIUM_copy_compressed_texture GL_CHROMIUM_copy_texture GL_CHROMIUM_lose_context GL_CHROMIUM_sync_query GL_EXT_EGL_image_external_wrap_modes GL_EXT_blend_func_extended GL_EXT_blend_minmax GL_EXT_color_buffer_float GL_EXT_color_buffer_half_float GL_EXT_debug_marker GL_EXT_discard_framebuffer GL_EXT_disjoint_timer_query GL_EXT_draw_buffers GL_EXT_draw_buffers_indexed GL_EXT_draw_elements_base_vertex GL_EXT_float_blend GL_EXT_frag_depth GL_EXT_instanced_arrays GL_EXT_map_buffer_range GL_EXT_multisampled_render_to_texture GL_EXT_occlusion_query_boolean GL_EXT_read_format_bgra GL_EXT_robustness GL_EXT_sRGB GL_EXT_shader_texture_lod GL_EXT_texture_compression_bptc GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc_srgb GL_EXT_texture_filter_anisotropic GL_EXT_texture_format_BGRA8888 GL_EXT_texture_norm16 GL_EXT_texture_rg GL_EXT_texture_storage GL_EXT_unpack_subimage GL_KHR_debug GL_KHR_no_error GL_KHR_parallel_shader_compile GL_KHR_robust_buffer_access_behavior GL_NV_EGL_stream_consumer_external GL_NV_fence GL_NV_pack_subimage GL_NV_pixel_buffer_object GL_OES_EGL_image GL_OES_EGL_image_external GL_OES_EGL_image_external_essl3 GL_OES_compressed_EAC_R11_signed_texture GL_OES_compressed_EAC_R11_unsigned_texture GL_OES_compressed_EAC_RG11_signed_texture GL_OES_compressed_EAC_RG11_unsigned_texture GL_OES_compressed_ETC2_RGB8_texture GL_OES_compressed_ETC2_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_RGBA8_texture GL_OES_compressed_ETC2_punchthroughA_sRGB8_alpha_texture GL_OES_compressed_ETC2_sRGB8_alpha8_texture GL_OES_compressed_ETC2_sRGB8_texture GL_OES_depth24 GL_OES_depth32 GL_OES_draw_buffers_indexed GL_OES_draw_elements_base_vertex GL_OES_element_index_uint GL_OES_fbo_render_mipmap GL_OES_get_program_binary GL_OES_mapbuffer GL_OES_packed_depth_stencil GL_OES_rgb8_rgba8 GL_OES_standard_derivatives GL_OES_surfaceless_context GL_OES_texture_border_clamp GL_OES_texture_float GL_OES_texture_float_linear GL_OES_texture_half_float GL_OES_texture_half_float_linear GL_OES_texture_npot GL_OES_texture_stencil8 GL_OES_vertex_array_object GL_OVR_multiview GL_OVR_multiview2 GL_WEBGL_video_texture
      WebGL 2 Extensions: EXT_color_buffer_float EXT_float_blend EXT_texture_compression_bptc EXT_texture_compression_rgtc EXT_texture_filter_anisotropic MOZ_debug OES_texture_float_linear OVR_multiview2 WEBGL_compressed_texture_s3tc WEBGL_compressed_texture_s3tc_srgb WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context
      Direct2D: true
      Target Frame Rate: 60
      DirectWrite: true (10.0.19041.1151)

      GPU #1
      Active: Yes
      Description: AMD Radeon RX 5700
      Vendor ID: 0x1002
      Device ID: 0x731f
      Driver Version: 27.20.22017.1004
      Driver Date: 7-9-2021
      Drivers: aticfx64 aticfx64 aticfx64 amdxc64 aticfx32 aticfx32 aticfx32 amdxc32 amdxn64 atidxx64 atidxx64 amdxn32 atidxx32 atidxx32 atiumdva atiumd6a
      Subsys ID: 2399148c
      RAM: 8176

      GPU #2
      Active: No
      RAM: 0

      Diagnostics
      AzureCanvasBackend: direct2d 1.1
      AzureCanvasBackend (UI Process): skia
      AzureContentBackend: skia
      AzureContentBackend (UI Process): skia
      AzureFallbackCanvasBackend (UI Process): skia
      CMSOutputProfile: AAAKQEFEQkUCEAAAbW50clJHQiBYWVogB98AAQATAA8AFwAUYWNzcE1TRlQAAAAAREVMTAAAAAAAAAAAAAAAAAAAAAAAAPbWAAEAAAAA0y1ERUxMyt219wiuwoIkftXmC6p3iAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKZGVzYwAAAPwAAACEY3BydAAAAYAAAAA4d3RwdAAAAbgAAAAUclhZWgAAAcwAAAAUZ1hZWgAAAeAAAAAUYlhZWgAAAfQAAAAUclRSQwAAAggAAAgMZ1RSQwAAAggAAAgMYlRSQwAAAggAAAgMY2hhZAAAChQAAAAsZGVzYwAAAAAAAAAqREVMTCBQMjcxNVEgQ29sb3IgUHJvZmlsZSwgRDY1MDAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAdGV4dAAAAABDb3B5cmlnaHQgKGMpIDIwMTUgRGVsbCBJbmMuAAAAAAAAAAAAAAAAAAAAAAAAAABYWVogAAAAAAAA9tYAAQAAAADTLVhZWiAAAAAAAAB8LgAAQAgAAAXSWFlaIAAAAAAAAFlSAACyogAAHcZYWVogAAAAAAAAIVcAAA1WAACvlWN1cnYAAAAAAAAEAAAAAAUACgAPABQAGQAeACMAKAAtADIANwA7AEAARQBKAE8AVABZAF4AYwBoAG0AcgB3AHwAgQCGAIsAkACVAJoAnwCkAKkArgCyALcAvADBAMYAywDQANUA2wDgAOUA6wDwAPYA+wEBAQcBDQETARkBHwElASsBMgE4AT4BRQFMAVIBWQFgAWcBbgF1AXwBgwGLAZIBmgGhAakBsQG5AcEByQHRAdkB4QHpAfIB+gIDAgwCFAIdAiYCLwI4AkECSwJUAl0CZwJxAnoChAKOApgCogKsArYCwQLLAtUC4ALrAvUDAAMLAxYDIQMtAzgDQwNPA1oDZgNyA34DigOWA6IDrgO6A8cD0wPgA+wD+QQGBBMEIAQtBDsESARVBGMEcQR+BIwEmgSoBLYExATTBOEE8AT+BQ0FHAUrBToFSQVYBWcFdwWGBZYFpgW1BcUF1QXlBfYGBgYWBicGNwZIBlkGagZ7BowGnQavBsAG0QbjBvUHBwcZBysHPQdPB2EHdAeGB5kHrAe/B9IH5Qf4CAsIHwgyCEYIWghuCIIIlgiqCL4I0gjnCPsJEAklCToJTwlkCXkJjwmkCboJzwnlCfsKEQonCj0KVApqCoEKmAquCsUK3ArzCwsLIgs5C1ELaQuAC5gLsAvIC+EL+QwSDCoMQwxcDHUMjgynDMAM2QzzDQ0NJg1ADVoNdA2ODakNww3eDfgOEw4uDkkOZA5/DpsOtg7SDu4PCQ8lD0EPXg96D5YPsw/PD+wQCRAmEEMQYRB+EJsQuRDXEPURExExEU8RbRGMEaoRyRHoEgcSJhJFEmQShBKjEsMS4xMDEyMTQxNjE4MTpBPFE+UUBhQnFEkUahSLFK0UzhTwFRIVNBVWFXgVmxW9FeAWAxYmFkkWbBaPFrIW1hb6Fx0XQRdlF4kXrhfSF/cYGxhAGGUYihivGNUY+hkgGUUZaxmRGbcZ3RoEGioaURp3Gp4axRrsGxQbOxtjG4obshvaHAIcKhxSHHscoxzMHPUdHh1HHXAdmR3DHeweFh5AHmoelB6+HukfEx8+H2kflB+/H+ogFSBBIGwgmCDEIPAhHCFIIXUhoSHOIfsiJyJVIoIiryLdIwojOCNmI5QjwiPwJB8kTSR8JKsk2iUJJTglaCWXJccl9yYnJlcmhya3JugnGCdJJ3onqyfcKA0oPyhxKKIo1CkGKTgpaymdKdAqAio1KmgqmyrPKwIrNitpK50r0SwFLDksbiyiLNctDC1BLXYtqy3hLhYuTC6CLrcu7i8kL1ovkS/HL/4wNTBsMKQw2zESMUoxgjG6MfIyKjJjMpsy1DMNM0YzfzO4M/E0KzRlNJ402DUTNU01hzXCNf02NzZyNq426TckN2A3nDfXOBQ4UDiMOMg5BTlCOX85vDn5OjY6dDqyOu87LTtrO6o76DwnPGU8pDzjPSI9YT2hPeA+ID5gPqA+4D8hP2E/oj/iQCNAZECmQOdBKUFqQaxB7kIwQnJCtUL3QzpDfUPARANER0SKRM5FEkVVRZpF3kYiRmdGq0bwRzVHe0fASAVIS0iRSNdJHUljSalJ8Eo3Sn1KxEsMS1NLmkviTCpMcky6TQJNSk2TTdxOJU5uTrdPAE9JT5NP3VAnUHFQu1EGUVBRm1HmUjFSfFLHUxNTX1OqU/ZUQlSPVNtVKFV1VcJWD1ZcVqlW91dEV5JX4FgvWH1Yy1kaWWlZuFoHWlZaplr1W0VblVvlXDVchlzWXSddeF3JXhpebF69Xw9fYV+zYAVgV2CqYPxhT2GiYfViSWKcYvBjQ2OXY+tkQGSUZOllPWWSZedmPWaSZuhnPWeTZ+loP2iWaOxpQ2maafFqSGqfavdrT2una/9sV2yvbQhtYG25bhJua27Ebx5veG/RcCtwhnDgcTpxlXHwcktypnMBc11zuHQUdHB0zHUodYV14XY+dpt2+HdWd7N4EXhueMx5KnmJeed6RnqlewR7Y3vCfCF8gXzhfUF9oX4BfmJ+wn8jf4R/5YBHgKiBCoFrgc2CMIKSgvSDV4O6hB2EgITjhUeFq4YOhnKG14c7h5+IBIhpiM6JM4mZif6KZIrKizCLlov8jGOMyo0xjZiN/45mjs6PNo+ekAaQbpDWkT+RqJIRknqS45NNk7aUIJSKlPSVX5XJljSWn5cKl3WX4JhMmLiZJJmQmfyaaJrVm0Kbr5wcnImc951kndKeQJ6unx2fi5/6oGmg2KFHobaiJqKWowajdqPmpFakx6U4pammGqaLpv2nbqfgqFKoxKk3qamqHKqPqwKrdavprFys0K1ErbiuLa6hrxavi7AAsHWw6rFgsdayS7LCszizrrQltJy1E7WKtgG2ebbwt2i34LhZuNG5SrnCuju6tbsuu6e8IbybvRW9j74KvoS+/796v/XAcMDswWfB48JfwtvDWMPUxFHEzsVLxcjGRsbDx0HHv8g9yLzJOsm5yjjKt8s2y7bMNcy1zTXNtc42zrbPN8+40DnQutE80b7SP9LB00TTxtRJ1MvVTtXR1lXW2Ndc1+DYZNjo2WzZ8dp22vvbgNwF3IrdEN2W3hzeot8p36/gNuC94UThzOJT4tvjY+Pr5HPk/OWE5g3mlucf56noMui86Ubp0Opb6uXrcOv77IbtEe2c7ijutO9A78zwWPDl8XLx//KM8xnzp/Q09ML1UPXe9m32+/eK+Bn4qPk4+cf6V/rn+3f8B/yY/Sn9uv5L/tz/bf//c2YzMgAAAAAAAQxEAAAF3///8yYAAAeUAAD9j///+6H///2iAAAD2wAAwHU=
      Display0: 3840x2160@60Hz : AMD Radeon RX 5700
      Display1: 2560x1440@60Hz : AMD Radeon RX 5700
      DisplayCount: 2
      HardwareStretching: both=0 window-only=0 full-screen-only=2 none=0 error=0
      GPUProcessPid: 7876
      ClearType Parameters: \\.\DISPLAY1 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ] \\.\DISPLAY2 [ Gamma: 1.8 Pixel Structure: RGB ClearType Level: 100 Enhanced Contrast: 50 ]

      Decision Log
      HW_COMPOSITING: available by default
      D3D11_COMPOSITING: available by default
      DIRECT2D: available by default
      D3D11_HW_ANGLE: available by default
      GPU_PROCESS: available by default
      WEBRENDER: available by default
      WEBRENDER_QUALIFIED: available by default
      WEBRENDER_COMPOSITOR: available by default
      WEBRENDER_PARTIAL: available by default
      WEBRENDER_SHADER_CACHE: available by default
      WEBRENDER_OPTIMIZED_SHADERS: available by default
      WEBRENDER_ANGLE: available by default
      WEBRENDER_DCOMP_PRESENT: available by default
      WEBRENDER_SOFTWARE: available by default
      OMTP: disabled by default: Disabled by default
      WEBGPU: disabled by default: Disabled by defaultblocked by runtime: WebGPU can only be enabled in nightly

    Crash Guard Disabled Features

    Workarounds

    Failure Log

  Media
     REMOVED
   
  Environment Variables

    MOZ_CRASHREPORTER_DATA_DIRECTORY: C:\Users\cbrou\AppData\Roaming\Thunderbird\Crash Reports
    MOZ_CRASHREPORTER_EVENTS_DIRECTORY: C:\Users\cbrou\AppData\Roaming\Thunderbird\Profiles\rpsb3f4t.default-release\crashes\events
    MOZ_CRASHREPORTER_PING_DIRECTORY: C:\Users\cbrou\AppData\Roaming\Thunderbird\Pending Pings
    MOZ_CRASHREPORTER_RESTART_ARG_0: C:\Program Files\Mozilla Thunderbird\thunderbird.exe
    MOZ_CRASHREPORTER_STRINGS_OVERRIDE: C:\Program Files\Mozilla Thunderbird\crashreporter-override.ini

  Important Modified Preferences

    accessibility.typeaheadfind.flashBar: 0
    browser.cache.disk.amount_written: 1516804
    browser.cache.disk.capacity: 1048576
    browser.cache.disk.filesystem_reported: 1
    browser.cache.disk.hashstats_reported: 1
    browser.search.region: DE
    extensions.lastAppVersion: 91.0
    font.internaluseonly.changed: true
    font.name.monospace.el: Consolas
    font.name.monospace.x-cyrillic: Consolas
    font.name.monospace.x-unicode: Consolas
    font.name.monospace.x-western: Consolas
    font.name.sans-serif.el: Calibri
    font.name.sans-serif.x-cyrillic: Calibri
    font.name.sans-serif.x-unicode: Calibri
    font.name.sans-serif.x-western: Calibri
    font.name.serif.el: Cambria
    font.name.serif.x-cyrillic: Cambria
    font.name.serif.x-unicode: Cambria
    font.name.serif.x-western: Cambria
    font.size.monospace.el: 14
    font.size.monospace.x-cyrillic: 14
    font.size.monospace.x-unicode: 14
    font.size.monospace.x-western: 14
    font.size.variable.el: 17
    font.size.variable.x-cyrillic: 17
    font.size.variable.x-unicode: 17
    font.size.variable.x-western: 17
    gfx.crash-guard.d3d11layers.appVersion: 78.12.0
    gfx.crash-guard.d3d11layers.deviceID: 0x731f
    gfx.crash-guard.d3d11layers.driverVersion: 27.20.22017.1004
    gfx.crash-guard.d3d11layers.feature-d2d: true
    gfx.crash-guard.d3d11layers.feature-d3d11: true
    gfx.crash-guard.status.d3d11layers: 2
    gfx.crash-guard.status.wmfvpxvideo: 2
    gfx.crash-guard.wmfvpxvideo.appVersion: 91.0
    gfx.crash-guard.wmfvpxvideo.deviceID: 0x731f
    gfx.crash-guard.wmfvpxvideo.driverVersion: 27.20.22017.1004
    idle.lastDailyNotification: 1628188190
    layers.mlgpu.sanity-test-failed: false
    media.gmp.storage.version.observed: 1
    media.hardware-video-decoding.failed: false
    network.trr.blocklist_cleanup_done: true
    places.database.lastMaintenance: 1628188190
    places.history.expiration.transient_current_max_pages: 37521
    privacy.purge_trackers.date_in_cookie_database: 0
    security.remote_settings.crlite_filters.checked: 1627997843
    security.remote_settings.intermediates.checked: 1627997843
    security.sandbox.content.tempDirSuffix: {f1c22d80-fb35-4d24-9490-02048e69cb95}
    security.sandbox.plugin.tempDirSuffix: {e982c1ac-0c31-4db5-9653-3ae4df4a4279}
    signon.importedFromSqlite: true
    storage.vacuum.last.index: 1
    storage.vacuum.last.places.sqlite: 1626789698
    ui.osk.debug.keyboardDisplayReason: IKPOS: Touch screen not found.

  Important Locked Preferences

  fission.autostart.session: false

  Places Database

  Accessibility

    Activated: false
    Prevent Accessibility: 0
    Accessible Handler Used: true
    Accessibility Instantiator:

  Library Versions

      Expected minimum version
      Version in use

      NSPR
      4.32
      4.32

      NSS
      3.68
      3.68

      NSSSMIME
      3.68
      3.68

      NSSSSL
      3.68
      3.68

      NSSUTIL
      3.68
      3.68

  Sandbox

    Content Process Sandbox Level: 0
    Effective Content Process Sandbox Level: 1
    Win32k Lockdown State for Content Process: Win32k Lockdown disabled -- Preference not set

  Startup Cache

    Disk Cache Path: C:\Users\cbrou\AppData\Local\Thunderbird\Profiles\rpsb3f4t.default-release\startupCache\startupCache.8.little
    Ignore Disk Cache: true
    Found Disk Cache on Init: false
    Wrote to Disk Cache: true

  Internationalization & Localization

      Application Settings
      Requested Locales: ["en-US"]
      Available Locales: ["en-US"]
      App Locales: ["en-US"]
      Regional Preferences: ["en-US"]
      Default Locale: "en-US"

      Operating System
      System Locales: ["en-US","fr-FR","de-DE"]
      Regional Preferences: ["en-US"]

  Remote Debugging (Chromium Protocol)

    Accepting Connections: false
    URL:

  Printing

  Modified print settings
     REMOVED
```

**Comment by RealRaven2000** [comment]
Yes, you're running 5.6.6 which isn't fully compatible yet with Tb91. can you test one of the new versions linked here, maybe 5.7 prerelease 120?

https://github.com/RealRaven2000/QuickFolders/issues/114#issuecomment-891362246

**Comment by RealRaven2000** [comment]
ok, it was something else. a function in Mailservices.copy has been renamed. Please try this fix:


[QuickFolders-mx-5.7pre174.zip](https://github.com/RealRaven2000/QuickFolders/files/6947954/QuickFolders-mx-5.7pre174.zip)

To install this preview, download the zip file and drag it into Thunderbird Add-ons manager.

**Comment by RealRaven2000** [comment]
> ok, it was something else. a function in Mailservices.copy has been renamed. Please try this fix:
> 
> [QuickFolders-mx-5.7pre174.zip](https://github.com/RealRaven2000/QuickFolders/files/6947954/QuickFolders-mx-5.7pre174.zip)
> 


It looks like there are some problems with "undo move message" in Thunderbird 91.0b6 - even though I supply a truthy `allowUndo` argument. Need to look into this one further.

**Comment by RealRaven2000** [comment]
Fixed in 5.7, [released on ATN](https://addons.thunderbird.net/thunderbird/addon/quickfolders-tabbed-folders/versions/5.7) today.
